bears/js/HappinessLintBear.py

```diff
@@ -20,5 +20,14 @@ class HappinessLintBear:
     CAN_DETECT = {'Syntax'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file):
+    def create_arguments(
+            filename, file, config_file,
+            use_spaces: bool = False):
+
+        if (use_spaces):
+            raise ValueError(
+                'Set "use_spaces = False" for HappinessLintBear'
+                'to avoid conflicts with other bears'
```

Comment:
That's not the reason :O The reason is that `happiness` just doesn't support space indentation.
(btw: you forgot to put a space between `HappinessLintBear` and `to avoid`^^)